src/components/newsletter.js

```diff
@@ -1,39 +1,39 @@
 import React from "react";
 
-import { patterns } from "./cloudImages";
+import { gallery, patterns } from "./cloudImages";
 import Mailchimp from "../components/mailchimp";
 
 function Newsletter() {
   return (
-    <section id="gallery" className="bg-white lg:flex-row px-4 lg:px-40 pt-12 lg:pt-24">
-      <div className="absolute left-0 fixed"></div>
+    <section id="gallery" className="bg-white lg:flex-row px-4 lg:px-40 pt-12 lg:pt-14">
+      <div className="absolute left-0 text-center">
+        <img className="-mt-8 lg:-mt-12 -pl-4" src={patterns.headerRec} alt="" />
```

Comment:
This is creating the formatting issue. Could you remove it?

Reply:
I have removed it and the footer is properly formatted now, thanks